src/View/Form/ContextInterface.php

```diff
@@ -21,6 +21,8 @@
  */
 interface ContextInterface
 {
+    public const VALID_ATTRIBUTES = ['length', 'precision', 'comment', 'null', 'default'];
```

Comment:
Wondering if this should be `protected`.

Reply:
I would start at protected and open up if we need to. Going from public to protected is harder.

Reply:
Err interface constant can be public only, so can't make it protected :)

Reply:
boooo